applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -152,6 +152,24 @@ impl AppState {
         Ok(())
     }
 
+    // Return alias or pub key if the contact is not in the list.
+    pub fn get_alias(&self, pub_key: &RistrettoPublicKey) -> String {
+        let pub_key_hex = format!("{}", pub_key);
+        // TODO: We can uncomment this to indicated unknown origin, otherwise there is our pub key.
```

Comment:
Self-spends are possible, so imported utxos are not the only case where this public key is our own.

Reply:
So something like "This wallet"? Or we can omit that completely, and who wants it can add him/herself as a contact.

Reply:
Yeah, I would prefer "Self" or "Me" to "Unknown"